dashboard/test/ui/features/teacher_tools/script_overview.feature

```diff
@@ -70,3 +70,10 @@ Feature: Unit overview page
     And I wait until element ".uitest-end-of-lesson-header:contains(You finished Lesson 3!)" is visible
     And I reload the page
     And  element ".uitest-end-of-lesson-header:contains(You finished Lesson 3!)" is not visible
+
+  Scenario: Unit overview lesson plan
```

Comment:
Can you add tests for the new lesson plans and student lesson plans as well? https://test-studio.code.org/s/allthemigratedthings?no_redirect=true should work for those tests

Reply:
ahhh yes the no_redirect is what I needed. Fixed the old one and added the two new ones! Thanks!